<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLabelsWire</class>
 <widget class="QWidget" name="FormLabelsWire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Этикетки для проволоки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton80_1016">
       <property name="text">
        <string>Этикетка 80*101,6 (пластик)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton95_110">
       <property name="text">
        <string>Этикетка 95*110 (пластик)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton45_70">
       <property name="text">
        <string>Этикетка 45*70</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton100_100">
       <property name="text">
        <string>Этикетка 100*100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
